CHANGES.txt

```diff
@@ -1,6 +1,53 @@
 Traits CHANGELOG
 ================
 
+DEV
```

Comment:
We'll eventually need a version number here. Are you planning to release directly from the master branch, or from a release branch? If a release branch, we can branch first, then insert the new version number here. If the master branch, then it may make sense to update the version number in this PR.

And then there's the obvious question: what version number? This is more than a bugfix release, so it at least deserves to be version 4.7.0. But there may be a case for 5.0.0 (in an effort to catch up with Traits UI).

Reply:
> One big reason to call it 5.0.0 is the complete removal of 2to3.

Yep, that's reason enough for me. Let's call it 5.0.0 then, unless there are any objections. I might ping the Slack channel to see if there are other opinions.